README.md

```diff
@@ -1,5 +1,10 @@
 <img src="scenarios/media/logo_cvbp.png" align="right" alt="" width="300"/>
 
+```diff
++ March 27: Released v1.1 with new and improved
++   functionality for image retrieval, object detection, 
```

Comment:
Could we add a bit more detail on what `new and improved functionality` refers to?

Maybe a subsection with more details?

Reply:
Actually, could we add a link here to the release page once it's updated?
ex) `For additional details on the release, please refer to our releases page.`